src/resources/LibraryStyles.css

```diff
@@ -393,6 +428,10 @@ input {
     cursor: pointer;
 }
 
+.SearchBar button.CancelButton:hover{
```

Comment:
This is also defined at line `659`, could be consolidated into a single definition. Nothing big though.

Reply:
this has been unified under the same block now, thanks!